packages/eslint-plugin-result/package.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "@sapphire/eslint-plugin-result",
+	"version": "0.1.0",
+	"description": "A TypeScript ESLint plugin for @sapphire/result",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"sideEffects": false,
+	"homepage": "https://github.com/sapphiredev/utilities/tree/main/packages/eslint-plugin-result",
+	"scripts": {
+		"test": "vitest run",
+		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
+		"build": "tsup && tsc -b src",
+		"prepack": "yarn build",
+		"bump": "cliff-jumper",
+		"check-update": "cliff-jumper --dry-run"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/sapphiredev/utilities.git",
+		"directory": "packages/eslint-plugin-result"
+	},
+	"files": [
+		"dist/**/*.js*",
+		"dist/**/*.mjs*",
+		"dist/**/*.d*"
+	],
+	"engines": {
+		"node": ">=v14.0.0",
+		"npm": ">=7.0.0"
+	},
+	"keywords": [
+		"@sapphire/eslint-plugin-result",
+		"bot",
+		"discord",
+		"lexure",
+		"promise",
+		"sapphire",
+		"standalone",
+		"try catch",
+		"ts",
+		"typescript",
+		"wrapper",
+		"yarn"
+	],
+	"bugs": {
+		"url": "https://github.com/sapphiredev/utilities/issues"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"devDependencies": {
+		"@favware/cliff-jumper": "^1.10.0",
+		"@vitest/coverage-c8": "^0.28.5",
+		"tsup": "^6.6.3",
+		"vitest": "^0.28.5"
```

Comment:
Maybe I'm misunderstanding rule tester but I don't see vitest being used in this package? If so, prune the config, deps and test script. 

On that note, how do the tests run right now anyway?

Reply:
It's still vitest, we need as a dep for CLI. `yarn test` runs the test files as usual, and the rule tester does uh.. magic? 

![](https://sucks-to-b.eu/jIvpOV.png)

Reply:
Ah okay!